### PR TITLE
Update README.md

### LINKED ISSUE #19: [README] Need to add `Community Forum`
**Description**

<a href="https://github.com/layer5io/service-mesh-labs/blob/master/README.md">README.md</a> is the first file one should read when starting a new project. Our community launched `Discussion forum` for help seekers; needs to publish it officially on README.md for convenience.

### Screenshots

**Current behaviour:**
image

**Desired behaviour:**
image

**Note:**
Need to add only last line for `Community Forum`

**Environment:**
 - OS: any [e.g. Ubuntu]
 - Browser: any [e.g. Chrome, Safari]
 - Version: any [e.g. 22]
 - Device: any [e.g. laptop, iPhone 8]

---
**Reference:**
- Take reference to this <a href="https://github.com/layer5io/layer5/pull/1949">pull request</a>

### PR DESCRIPTION
**Pull request by Patil2099**
Fixes #19

### CODE REVIEW

**Review by includestdlibrary — CHANGES_REQUESTED** [review]
Thanks @Patil2099 for the PR. 
Here are some instructions to make sure that your commits are signed off- https://github.com/layer5io/service-mesh-labs/pull/20/checks

File: README.md [hunk 1]
```diff
@@ -28,7 +28,8 @@ Find out more on the <a href="https://layer5.io/community">Layer5 community</a>.
 <p>
 ✔️ <em><strong>Join</strong></em> any or all of the weekly meetings on <a href="https://calendar.google.com/calendar/b/1?cid=bGF5ZXI1LmlvX2VoMmFhOWRwZjFnNDBlbHZvYzc2MmpucGhzQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20">community calendar</a>.<br />
 ✔️ <em><strong>Watch</strong></em> community <a href="https://www.youtube.com/playlist?list=PL3A-A6hPO2IMPPqVjuzgqNU5xwnFFn3n0">meeting recordings</a>.<br />
-✔️ <em><strong>Access</strong></em> the <a href="https://drive.google.com/drive/u/4/folders/0ABH8aabN4WAKUk9PVA">community drive</a>.<br />
+✔️ <em><strong>Access</strong></em> the <a href="https://drive.google.com/drive/u/4/folders/0ABH8aabN4WAKUk9PVA">community drive</a>.<br />\
```
Comment (amino19) [inline README.md:31]: @Patil2099 : The slash `\` at last, need to be remove that, & please signed-off your commits :slightly_smiling_face: 

& yeah, you're good to go :rocket:

**Comment by leecalcote** [comment]
@Patil2099 checking in... will you be returning to amend your commit with a sign-off?

File: README.md (hunk 1 above)
Comment (leecalcote) [inline README.md:31]:
```suggestion
✔️ <em><strong>Access</strong></em> the <a href="https://drive.google.com/drive/u/4/folders/0ABH8aabN4WAKUk9PVA">community drive</a>.<br />
```

**Review by leecalcote — APPROVED** [review]
Thank you, @amino19. I remove the backslash.

**Pull request merged**